src/main.rs

```diff
@@ -13,6 +13,19 @@ mod expander;
 mod hippo_notifier;
 mod hippofacts;
 
+/// Indicate which flags are required for bindle builds
+#[allow(dead_code)]
+enum BindleBuildRequirements {
+    /// A Bindle server URL is required
+    RequireBindleServer,
+    /// An explicit stage directory is required
+    RequireStageDirectory,
+    /// Both the Bindle server and the stage directory are required
```

Comment:
We don't have any use for these last two yet...?

Reply:
Wondering if `struct BindleBuildRequirements` would be more convenient both for packaging and for fieldwise checking?  Having to check combinatoric sets of enum values is setting off my spidey sense.  #SpideySenseDrivenDevelopment

Reply:
I feel like I am trying to guess what you will like and won't like. I literally left the last two because I assumed you would say, "But if you don't include these, then you are indicating to the API user that the params are mutually exclusive."

Reply:
Okay, so clearly, the issue is that we are misusing Clap at this point. As I mentioned before, what we _should_ be doing is switching to Clap's struct-based stuff, which I have been working on in a separate PR. 

All I am trying to do here is fix the issue so that it is not broken today, and not block this fix by a massive refactor. Sure, we could do a builder pattern. Or a struct. Or all kinds of things... but I am gonna refactor all of that out anyway.